.github/workflows/csharp-ci.yaml

```diff
@@ -1,9 +1,9 @@
 name: CI for C# Toolkit Common Telemetry
 on:
     push:
-        branches: [master]
+        branches: [main, master]
```

Comment:
Will there be a second PR to come back and remove the "master" branch?

Reply:
Yes